ENIGMAsystem/SHELL/Universal_System/Resources/AssetArray.h

```diff
@@ -121,11 +121,11 @@ class AssetArray {
 
   AssetArray() {}
 
-  iterator begin() { return ++iterator(*this, -1); }
-  iterator end() { return iterator(*this, size()); }
+  iterator begin() { return ++iterator(*this, -(LEFT + 1)); }
+  iterator end() { return iterator(*this, size() - LEFT); }

```

Comment:
I don't know if this is correct for the iterator or not. Do we want the iterator for looping all the assets to also go over the left values too?